File: VisitBosnia/visit_bosnia_mobile/lib/model/review/review_search_object.dart
class ReviewSearchObject {
  String? searchText;
  bool? includeTouristFacility;
  bool? includeAppUser;
  int? agencyId;
  int? facilityId;
  int? rating;

  ReviewSearchObject(
      {this.searchText,
      this.includeTouristFacility,
      this.includeAppUser,
      this.agencyId,
      this.facilityId,
      this.rating});

  ReviewSearchObject.fromJson(Map<String, dynamic> json) {
    searchText = json['searchText'];
    includeTouristFacility = json['includeTouristFacility'];
    includeAppUser = json['includeAppUser'];
    agencyId = json['agencyId'];
    facilityId = json['facilityId'];
    rating = json['rating'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['searchText'] = this.searchText;
    data['includeTouristFacility'] = this.includeTouristFacility;
    data['includeAppUser'] = this.includeAppUser;
    data['agencyId'] = this.agencyId;
    data['categoryId'] = this.facilityId;
    data['cityId'] = this.rating;
    return data;
  }
}

File: VisitBosnia/VisitBosnia.Model/SearchObjects/ReviewSearchObject.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace VisitBosnia.Model.SearchObjects
{
    public class ReviewSearchObject
    {
        public string? SearchText { get; set; }
        //public int? Rating { get; set; }
        public int? AgencyId { get; set; }
        public bool? IncludeTouristFacility { get; set; }
        public bool? IncludeAppUser { get; set; }
        public int? Rating { get; set; }

        public int? FacilityId { get; set; }

    }
}
